pulsar-transformations/src/test/java/com/datastax/oss/pulsar/functions/transforms/TransformFunctionTest.java

```diff
@@ -157,6 +203,48 @@ void testDropFields() throws Exception {
     assertNull(valueAvroRecord.getSchema().getField("valueField3"));
   }
 
+  @Test
+  void testComputeFields() throws Exception {
+    String userConfig =
+        (""
+                + "{'steps': ["
+                + "    {'type': 'compute-fields', 'fields':["
+                + "        {'name': 'newField1', 'expression' : '5*3', 'type': 'INT32', 'part' : 'key'},"
+                + "        {'name': 'newField2', 'expression' : 'value.valueField1.toUpperCase()', 'type': 'STRING', 'part' : 'key'},"
```

Comment:
Generally I don't think it is a good idea to advertise that you can call Java methods.
I am sure that people will do it.

If there is some way to prevent it really would prefer, otherwise we will have probably JDK compatibility issues (if they remove something.....expressions will stop to work)

if we can't stop it, then I would simply do not document it.

keeping it here in the example works for me

Reply:
it should be possible to disable method invocations. Let me try it along with a utility method as a proof concept that we don't need the first, and we can provide users with the latter...